### ISSUE TITLE
发现一个bug

### ISSUE THREAD

**Issue by zysen0709** [YanxinTang/clipboard-online#29]
电脑复制后可以推送到ipad，此时正常，然后电脑再复制另一个东西后，ipad用快捷指令后还是获取到上一次复制的东西，并且此时电脑的粘贴板被清空了,求救~~

**Comment by zysen0709** [comment]
两次都是复制图片

**Comment by YanxinTang** [comment]
先重启再试一下。确实有这个问题，但是我之前也没找到是啥情况，年后我再试试。

**Comment by zysen0709** [comment]
大佬，是更新了吗，能不能发个exe呢

**Comment by YanxinTang** [comment]
没有，更新都是在 github release 页面。

**Comment by zysen0709** [comment]
作者，关于这个bug发现一个现象就是，如果每次连续复制的图片比较小的情况，就不会出现bug，若连续复制大的图片(截图区域较大)的时候就会出现bug

**Comment by YanxinTang** [comment]
好的，我试试

**Comment by YanxinTang** [comment]
> ipad用快捷指令后还是获取到上一次复制的东西

有些情况下复制图片确实会粘贴不出来，但是在捷径中用快速查看打出来又确实也收到了图片，暂时不能确定哪里出了问题

> 此时电脑的粘贴板被清空了

 我测试时没有发现，有没有可能是误操作了？

**Comment by zysen0709** [comment]
[链接：https://pan.baidu.com/s/1mcQOwyYFx6rMNX1c_Hp7Rg 
提取码：t4de 
--来自百度网盘超级会员V3的分享](url)

**Comment by zysen0709** [comment]
这是我录的视频  麻烦看看啦

**Comment by YanxinTang** [comment]
你视频中的这些操作我试了还是没法稳定复现。

不过最后复制新内容之后剪切板也无法粘下来这种情况我之前也偶尔碰到，不过一直也没找到复现的方法。你重启一下程序还能复现吗？

**Comment by zysen0709** [comment]
是的 能稳定复现在我这  解决办法是重置剪切板 要么从ipad推送到windows  或者windwos复制文字也可以没这个bug  就是连续大图会有bug

**Comment by zysen0709** [comment]
对了 连续大图后  再在windows上复制东西（也就是第三次）  复制后在windwos上都无法粘贴  第四五次还是会如此  这时得退出电脑端的软件或者用ipad推送到win来解决

**Comment by zysen0709** [comment]
貌似发现了 是ipad获取了大截图后， 此时win的粘贴板就失效了，无论是文字图片都复制不了

**Comment by zysen0709** [comment]
是的 能稳定复现在我这 总结一下应该是这样
1.ipad获取了大截图后， 此时win的粘贴板就失效了，无论是文字图片都复制不了  复制任何东西在自己电脑上都粘贴不上 此时要么重启软件解决 要么 ipad推送到win来恢复剪切板正常功能
2.我刚才误打误撞从windows复制一个50MB的MP4文件到ipad上  之后ipad可以连续截图复制文字等操作  不会有问题出现   但过一段时间bug又自动出现了  也就是1处所描述的

**Comment by YanxinTang** [comment]
> 无论是文字图片都复制不了

我猜测是剪切板没有被正常关闭导致的，不过説来奇怪，我这边很难复现（按你的方法也是），只是之前用的时候偶尔会碰到。我会继续尝试解决这个问题。

**Comment by Poppy-u** [comment]
为什么一重启软件就打不开了，关闭了win10的安全中心都还是打不开

**Comment by YanxinTang** [comment]
看下日志？

Poppy-u ***@***.***> 于2022年3月21日周一 08:24写道：

**Comment by Poppy-u** [comment]
> 

应该是我的安全中心没关闭彻底，我用了一个小工具彻底关闭安全中心后就可以正常使用了，总是误报我的软件
